src/IOCapture.jl

```diff
@@ -98,7 +102,8 @@ function capture(
     rethrow::Type=Any,
     color::Bool=false,
     passthrough::Bool=false,
-    capture_buffer=IOBuffer()
+    capture_buffer=IOBuffer(),
+    io_context::Vector=[],
 )
```

Comment:
Since the type is broad, maybe we could add some error checks, if they make sense:

1. Elements must all the `Pair`s?
2. `first` of each element must be a `Symbol`?

They should throw immediately on 117 anyway, but with `MethodError`s, so we could give more helpful errors here to the user.